instrumentation/jaxrs-client/jaxrs-client-2.0-testing/src/test/groovy/JaxRsClientTest.groovy

```diff
@@ -44,6 +44,8 @@ abstract class JaxRsClientTest extends HttpClientTest<Invocation.Builder> implem
     try {
       def body = BODY_METHODS.contains(method) ? Entity.text("") : null
       def response = request.build(method, body).invoke()
+      // read response body
+      response.readEntity(String)
       try {
         response.close()
```

Comment:
Guessing close isn't enough? Maybe just need to add it to the other code path, generally prefer to close over reading a response to nothing if it's possible

Reply:
As far as I understand the problem with just closing is that server might not have completed sending data. If that is the case then closing will cause an error (broken pipe) at the server which I'd like to avoid to see whether this also affects the flakiness.